### PR TITLE
AC-307 Bidirectional Text

### PR DESCRIPTION
**Pull request by hao555sky**

### CODE REVIEW

**Comment by hao555sky** [comment]
Why the checks are failed with the same error. It said 

> What went wrong:
A problem occurred configuring project ':openmrs-client'.
> You have not accepted the license agreements of the following SDK components:
[Android SDK Build-Tools 25.0.2].
Before building your project, you need to accept the license agreements and complete the installation of the missing components using the Android Studio SDK Manager.
Alternatively, to learn how to transfer the license agreements from one workstation to another, go to http://d.android.com/r/studio-ui/export-licenses.html
> Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
BUILD FAILED

How to solve it? In fact, I have no experience.

File: openmrs-client/gradle.properties [hunk 1]
```diff
@@ -12,8 +12,8 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-#Fri May 27 14:16:33 IST 2016
+#Fri Mar 10 20:44:25 CST 2017
 systemProp.https.proxyPort=8080
-systemProp.http.proxyHost=10.3.100.207
+systemProp.http.proxyHost=proxy.buptnsrc.com
```
Comment (AvijitGhosh82) [inline openmrs-client/gradle.properties]: Keep this file out of the git repo please. The local proxy settings should not be inflicted on others.

File: openmrs-client/build.gradle [hunk 2]
```diff
@@ -20,7 +20,7 @@ def version = "2.6.1-beta"
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    buildToolsVersion '25.0.2'
```
Comment (AvijitGhosh82) [inline openmrs-client/build.gradle]: Don't change this for now.

**Comment by hao555sky** [comment]
How about now? @AvijitGhosh82

**Pull request merged**